tests/core/test_expectations.py

```diff
@@ -2105,7 +2179,7 @@ def test_with_expectations(input_df,
     # Decorate the mock function with required args
     @se.with_expectations(
         "dq_spark.test_final_table",
-        user_conf={user_config.se_notifications_on_fail: False},
+        user_conf={user_config.se_notifications_on_fail: False, user_config.se_dq_rules_params: {'table' : 'test_table'} },
```

Comment:
Can we also test a negative scenario which should raise exception, when you don't pass user_config in this use_case!

Reply:
added negative test scenario